### DISCUSSION TITLE
findAllActiveListingsByShop api not working

### DISCUSSION THREAD

**Discussion by samikoja** [etsy/open-api#926] [Q&A]
findAllActiveListingsByShop api not working after migrate from v2 to v3 , it return status 200, but the response is {count: 0, result:[]} while I have an active products on my store. if we remove **/active** it is working same as should work with **/active**

**Comment by PMStanley** [comment]
If you remove /active you're actually calling a different endpoint (which also has different auth requirements): https://developer.etsy.com/documentation/reference/#operation/getListingsByShop
If it's just your shop you need access to and getListingsByShop is working fine then you may want to just stick with that endpoint.

However, to the point of findAllActiveListingsByShop not working for you I've just tested and it is working fine for me with just my API key, I'm happy to test again if you let me know what shopID isn't working for you.

**Reply by samikoja** [reply]
@PMStanley thank you for your response, honestly it is wired because when my store is installed as V3 it working fine, but when I test it on store already installed using V2 api and migrate it to V3 it doesn't work, I don't know if this is the reason but these what happen with me
shopID is 16081005

**Reply by PMStanley** [reply]
@samikoja same result for me with that shopID. I can see it should have 11 active listings but 0 are being returned by findAllActiveListingsByShop

Definitely one to take up with Etsy support via email.

**Reply by benjymouse** [reply]
Same here, for different shop IDs. FYI @nateswart - looks like a bug

**Reply by nateswart** [reply]
We will investigate this

**Reply by emergingdzns** [reply]
We are experiencing the same issues. One shop if you query for active listings it only gives us 10 of their active listings. But if we query as the shop with their token we get all 47. Shop id is 10037960.

**Reply by itiievskyi** [reply]
> We are experiencing the same issues. One shop if you query for active listings it only gives us 10 of their active listings. But if we query as the shop with their token we get all 47. Shop id is 10037960.

Interesting. I still see that this shop has 47 active listings, but I'm able to get 20 of them calling `/active`, not 10. Seems like something changed for another 10 listings.

**Comment by gosixl** [comment]
Me too getting nothing ({count: 0, result:[]}) despite all listings being active if I check with getListingsByShop

`   
    public static function search_for_product($uid, $string, $n=10) {
        
        $headers = [
            "Authorization: Bearer " . self::get_token($uid),
            "x-api-key: " . ETSY_CLIENT_ID,
            "Accept: application/json",
        ];
        
        // Build query parameters
        $params = [
            'keywords' => $string,
            'limit' => $n,
        ];
        
        $shop_id = get_user_meta($uid, 'ext_store_id_for_etsy', true);
        $url = "https://openapi.etsy.com/v3/application/shops/{$shop_id}/listings/active?" . http_build_query($params);

        $response = http_request("GET", $url, [], $headers, true);
        if ($response['code'] != 200) {
            custom_log(['Error fetching products from Etsy', $response]);
            return [];
        }`
